src/libertem_live/udf/monitor.py

```diff
@@ -13,21 +15,80 @@ class SignalMonitorUDF(UDF):
     individual merge steps. The end result depends on the merge order.
     '''
 
+    def get_backends(self):
+        return (
+            backend for backend in self.BACKEND_ALL
+            if backend not in {self.BACKEND_CUPY_SCIPY_COO, self.BACKEND_SCIPY_COO}
+        )
```

Comment:
This needs to be a list comprehension, as generators can't be pickled:

```suggestion
        return [
            backend for backend in self.BACKEND_ALL
            if backend not in {self.BACKEND_CUPY_SCIPY_COO, self.BACKEND_SCIPY_COO}
        ]
```